specs/agents/tracing-instrumentation-db.md

```diff
@@ -227,7 +227,7 @@ The Elasticsearch cluster name is not always available in ES clients, as a resul
 | MySQL                | `mysql`     |
 | MariaDB              | `mariadb`   |
 | PostgreSQL           | `postgresql`|
-| Microsoft SQL server | `sqlserver` |
+| Microsoft SQL server | `mssql`     |
```

Comment:
REVIEW NOTE: I think this use of `sqlserver` was unintended. "json-specs/span_types.json" suggests that "sqlserver" is deprecated in favour of "mssql".